contributors/devel/help-wanted.md

```diff
@@ -2,12 +2,91 @@
 
 Items marked with the `help wanted` label need to ensure that they are:
 
-- Sufficiently actionable: clear description of what should be done
-- Tractable for new/casual contributors: there is documentation how that type of change should be made
-- Goldilocks priority: Not too high that a core contributor should do it, but not too low that it isn't useful enough for a core contributor to spend time to review it, answer questions, help get it into a release, etc.
-- Up to date: Often these issues become obsolete and have already been done, are no longer desirable, no longer make sense, change in priority, change in difficulty, etc.
+- **Low Barrier to Entry**<br/>
```

Comment:
I think we can remove the <br/> in the end?

Reply:
oops, I mean `</br`>

Reply:
oops! My markdown previewer was giving me grief, but github renders it just fine without. Thanks for noticing that this could be cleaned up!